addons/scopes/functions/fnc_inventoryCheck.sqf

```diff
@@ -82,8 +82,8 @@ private _newOptics = [_player] call FUNC(getOptics);
 private _newGuns = [primaryWeapon _player, secondaryWeapon _player, handgunWeapon _player];
 {
     if ((_newOptics select _x) != (GVAR(Optics) select _x) || (_newGuns select _x != GVAR(Guns) select _x)) then {
-        GVAR(baseAngle) set [_x, [_player, _x, _newGuns select _x, _newOptics select _x] call FUNC(getBaseAngle)]; 
```

Comment:
I think we only wanted to change the `getBoreHeight` line; `getBaseAngle` still expects 4 args
Now getting error when shooting
```
  Error position: <vectorAdd [0, 0, _zeroCorrection - _base>
  Error Type Number,Not a Number, expected Number
File z\ace\addons\scopes\functions\fnc_firedEH.sqf, line 32
Error in expression <aseAngle];
} else {
_zeroing = _zeroing vectorAdd [0, 0, _zeroCorrection - _base>
```